iis-schemas/src/main/avro/eu/dnetlib/iis/importer/ProjectToOrganization.avdl

```diff
@@ -0,0 +1,10 @@
+@namespace("eu.dnetlib.iis.importer.schemas")
+protocol IIS{
+
+    record ProjectToOrganization {
```

Comment:
This is the convention we followed in importer model (see `DocumentToProject`, `DocumentToMDStore`, `DocumentToDocumentClasses`, `DocumentToDataSet`, `DocumentToConceptId`, `DocumentToConceptIds`). 

We could discuss all the implications and eventually change this convention. But I will prefer to do this as a separate task. Until then let's keep it aligned.